projects/ontimize-web-ngx/src/lib/components/table/extensions/skeleton/o-table-skeleton.component.spec.ts

```diff
@@ -8,31 +9,36 @@ import { appConfigFactory } from '../../../../services/app-config.provider';
 import { AppearanceService } from '../../../../services/appearance.service';
 import { AuthService } from '../../../../services/auth.service';
 import { MatDialogModule } from '@angular/material/dialog';
+import { LocalStorageService } from '../../../../services/local-storage.service';
+import { NgxSkeletonLoaderModule } from 'ngx-skeleton-loader';
 
 describe('OTableSkeletonComponent', () => {
   let component: OTableSkeletonComponent;
   let fixture: ComponentFixture<OTableSkeletonComponent>;
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      imports: [MatDialogModule],
+      imports: [MatDialogModule, NgxSkeletonLoaderModule],
```

Comment:
why all these changes of NgxSkeletonLoaderModule into this PR?

Reply:
Because the tests failed because the localstorageservice had to be added and the warning also came out that it did not recognize the ngx-skeleton-loadertag